.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE] - Requested feature"
+labels: 'feature'
```

Comment:
This should be 'enhancement', not 'feature'.

Reply:
Done.